boards/pba-d-01-kw2x/Makefile.dep

```diff
@@ -0,0 +1,4 @@
+ifneq (,$(filter ng_netif,$(USEMODULE)))
+  USEMODULE += kw2xrf
```

Comment:
so, we should add @gebart s snip to the RIOT/Makefile.dep? That would make this PR dependant of #2947. But the ```boards/pba-d-01-kw2x/Makefile.dep``` can remain as is?